rules/set-procmount-default/raw.rego

```diff
@@ -0,0 +1,98 @@
+package armo_builtins
+
+# Fails if container does not define the "procMount" parameter as "Default"
+deny[msga] {
+    # checks at first if we the procMountType feature gate is enabled on the api-server
+	obj := input[_]
+	is_control_plane_info(obj)
```

Comment:
[question]: Is there no need to check the api-server pod in this case if we are checking the control plane info?

Reply:
Yes, you are right.
I'm providing a check for this.